packages/trader/build/webpack.config-test.js

```diff
@@ -12,7 +12,10 @@ module.exports = function (env, argv) {
         externals   : [nodeExternals()],
         mode        : IS_RELEASE ? 'development' : 'production',
         module      : {
-            rules: rules(true),
+            rules: rules(
+                true,
+                (env === 'mocha_only')
```

Comment:
Does this type of `env` argument passing not break the `env.base` usage on line 6?